doc/versionHistory.rst

```diff
@@ -6,6 +6,14 @@
 Version History
 ##################
 
+.. _lsst.ts.wep-14.11.0:
```

Comment:
Should just be a minor update to 14.10.1 I think since this is just a fix of existing functionality.